### ISSUE TITLE
Update RSS feed generator

### ISSUE THREAD

**Issue by mooreds** [FusionAuth/fusionauth-site#1480]
Right now the RSS feed generator depends on two sources of data. The account.fusionauth.io API list and then the fusionauth.io release notes.

Unfortunately, the second data source should be the local instance, not the prod instance.

**Comment by mooreds** [comment]
Hmmm. We can't refer to localhost because we are building before the server is started, and I'm not sure how to access the release notes page before then (maybe it is available via one of the https://jekyllrb.com/docs/variables/ variables?).

**Comment by robotdan** [comment]
Our local dev environment is configured to proxy to docs via `https://site-local.fusionauth.io` - so you should not need to have `jekyll` running to access the release notes - but we will want to have them compiled before we gen the latest version.

**Comment by mooreds** [comment]
Yeah, sorry for being imprecise, this is an issue in the github action, which is how the website gets generated. The issue is with how I implemented the RSS feed generation. It happens in a generator, which occurs before the site is available. 

The current steps are:

* start building the local site in the github action (from `master`).
* the RSS feed reaches out to fusionauth.io and grabs the release notes, builds the rss feed file.
* the site completes building
* the site is synced to s3

The issue is with step 2. We really want the local file, but we can't reach out to it and grab it until the local server is up and running. So I need to either

* move the RSS feed generation step after the local server completes
* accept that the RSS feed will always be generated the prod push after it is available in the release notes (the current state of affairs)
* process the release notes as asciidoc rather than HTML (which would let me do it in that first step, because the file should be available).
